src/pip/_internal/operations/prepare.py

```diff
@@ -498,13 +492,11 @@ def prepare_linked_requirement(
                     logger.info('Link is a directory, ignoring download_dir')
                 elif local_file:
                     download_location = os.path.join(
-                        download_dir, link.filename
-                    )
+                        download_dir, link.filename)
```

Comment:
```suggestion
                        download_dir, link.filename
                    )
```

Reply:
I understand that this is to clean up the way for black and black is an automation (i.e. non-human) but ugh!  I promise that this is my last complain on pip's styling decision though :smile: